.github/workflows/cve-scanning-gradle.yml

```diff
@@ -0,0 +1,25 @@
+name: CVE Scanning for Gradle
+
+on:
+  push:
+    paths:
+      - '**/build.gradle'
+      - 'allow-list.xml'
+      - '.github/workflows/cve-scanning-gradle.yml'
+
+
+jobs:
+  scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+      - name: Build with Gradle
+        run: ./gradlew build
+      - name: CVEs
+        run: ./gradlew dependencyCheckAnalyze
```

Comment:
I think that '_dependencyCheckAggregate_' should be used rather to scan all modules. 
Running dependencyCheckAnalyze showed in the report **_"Dependencies Scanned: 0 (0 unique)"_** while with '_dependencyCheckAggregate_' I can see **_'Dependencies Scanned: 151 (151 unique)'_**

Reply:
yes, in multi-module project, Aggregate should be the one to use.

Reply:
Thanks @symphony-soufiane  and @yinan-symphony! I've changed it in the PR. As you can see there are a [few CVEs.](https://github.com/TheJuanAndOnly99/symphony-bdk-java/actions/runs/3531749359/jobs/5925266182) Can you please take a look. If they are safe to ignore you can use the [allow-list.xml file](https://github.com/finos/security-scanning/blob/main/allow-list.xml) to do so.